pnpm-lock.yaml

```diff
@@ -1,7 +1,7 @@
 lockfileVersion: '9.0'
 
 settings:
-  autoInstallPeers: true
+  autoInstallPeers: false
```

Comment:
What's this change? Is this what's causing the amp package to no longer have kit as a dependency in the lockfile? Is that a problem?

Reply:
the default value for that setting was changed from false to true with pnpm 8.0